src/mem3_shards.erl

```diff
@@ -242,7 +242,49 @@ terminate(_Reason, #st{changes_pid=Pid}) ->
     exit(Pid, kill),
     ok.
 
-code_change(_OldVsn, #st{}=St, _Extra) ->
+code_change(1 = OldVsn, {st, MaxSize, CurSize, ChangesPid} = St, _Extra) ->
+    twig:log(notice, "~p code_change ~p ~p", [?MODULE, OldVsn, St]),
```

Comment:
It was helpful for debugging, and since it should only happen once, I figured I might as well leave it in. Do you want me to remove the logging?